docs/docs/graphql.md

```diff
@@ -27,8 +27,8 @@ Which returns this:
 }
 ```
 
-Notice how the query signature exactly matches the returned JSON signature. This is possible because in GraphQL, you query against a `schema` that is the representation of your available data. Don't worry about where the schema comes from right now, Gatsby takes care of organizing all of your data for you and making it discoverable with a tool called GraphiQL. GraphiQL is a UI that lets you 1) run queries against your data in the browser, and 2) dig into the structure of data available to you through a data type explorer.
+Note como o formato da consulta tem exatamento o mesmo formato dos dados retornados em JSON. Isso é possivel por que em GraphQL, a consulta é comparada com um modelo de dados, `schema`, que é a representação dos dados disponíveis. Não se preocupe ainda sobre de onde vem o modelo de dados. O Gatsby organiza todos os seus dados para você e os torna acessíveis através de uma ferramenta chamada GraphiQL. GraphiQL e uma interface gráfica que permite que você 1) faça consultas em seus dados através do navegador, e 2) investigue a estrutura dos dados disponiveis a você através de um navegador de tipo de dados.
 
-If you want to know more about GraphQL, you can read more about [why Gatsby uses it](/docs/why-gatsby-uses-graphql/) and check out this [conceptual guide](/docs/querying-with-graphql/) on querying data with GraphQL.
+Se você quiser saber mais sobre GraphQL, você pode ler sobre o [porquê do Gatsby usar isso](/docs/why-gatsby-uses-graphql/), e conferir o [guia conceitual](/docs/querying-with-graphql/) sobre a consulta de dados utilizando GraphQL.
```

Comment:
Sugestão: why Gatsby uses it  ->  porque o o Gatsby adotou essa tecnologia

on querying data with ->  como consultar dados usando...

Reply:
```suggestion
Se você quiser saber mais sobre GraphQL, você pode ler sobre [porque o Gatsby adotou essa tecnologia](/docs/why-gatsby-uses-graphql/), e conferir o [guia conceitual](/docs/querying-with-graphql/) sobre como consultar dados utilizando o GraphQL.
```